src/app/utils.ts

```diff
@@ -94,8 +94,8 @@ export const assertDocumentElement = (message: string = 'BUG: Document Element i
 };
 
 export const scrollTo = (elem: Element | string) => {
-  const html = assertDocumentElement();
-  const padding = assertWindow().getComputedStyle(html).getPropertyValue('scroll-padding-top');
```

Comment:
@Dantemss i really really liked this approach, but had to re-think it because edge doesn't support scroll-padding at all so this returns empty all the time